types/email-templates/email-templates-tests.ts

```diff
@@ -1,10 +1,11 @@
+/// <reference lib="dom" />
 import Email = require('email-templates');
 import { createTransport } from 'nodemailer';
 import path = require('path');
 
 const locals = {
     locale: 'en',
-    name: 'Elon',
+    name: 'Edward',
 };
```

Comment:
Why change the name?

Reply:
Some IDEs extensions for spell checks (in literals) mark 'Elon' as an error (unless added to known words). Sounds a bummer, but fact of the life. I can retract this line change, though, let me know:
https://github.com/streetsidesoftware/vscode-spell-checker

Reply:
Was just curious, it's fine. Maybe we change it back later.

Reply:
I don't think it should be here. The commit is called "remove direct 'dom' dependency" and this is unrelated to removing dom as dependency. It also depends on a specific IDE with a very specific extension.